core/win32/syscall.c

```diff
@@ -1744,7 +1744,10 @@ propagate_options_via_env_vars(dcontext_t *dcontext, HANDLE process_handle,
      * model without requiring setting up config files for children.
      */
     uint64 peb;
-    bool peb_is_32 = is_32bit_process(process_handle);
+    bool peb_is_32 = is_32bit_process(process_handle)
+	    // if x64 client targeting WOW64 we need to 
```

Comment:
nit: Modern style is to prefer capitalized and punctuated whole-sentence comments.